### PR TITLE
felangel/master

### PR DESCRIPTION
**Pull request by ThinkDigitalSoftware**

### CODE REVIEW

**Review by felangel — CHANGES_REQUESTED** [review]
Thanks for the contribution! Can you please add some tests to get 100% coverage?

**Comment by ThinkDigitalSoftware** [comment]
Honestly I don't understand how it's not still 100% coverage since I didn't
create any new methods. I'm not too familiar with tests and coverage

File: lib/src/hydrated_bloc_storage.dart [hunk 1]
```diff
@@ -36,8 +37,12 @@ class HydratedBlocStorage implements HydratedStorage {
 
     if (await file.exists()) {
       try {
-        storage =
-            json.decode(await file.readAsString()) as Map<String, dynamic>;
+        String source = await file.readAsString();
+        if (source.isEmpty) {
+          await file.delete();
```
Comment (felangel) [inline lib/src/hydrated_bloc_storage.dart:42]: why do we need to delete the file when source is empty?

**Comment by felangel** [comment]
Closing this for now and am addressing your feedback as part of https://github.com/felangel/hydrated_bloc/pull/16 👍

File: lib/src/hydrated_bloc_storage.dart (hunk 1 above)
Comment (ThinkDigitalSoftware) [reply lib/src/hydrated_bloc_storage.dart:42]: I think it was there as cleanup.

**Pull request closed without merging**